### PR TITLE
0.0.2

### PR DESCRIPTION
**Pull request by at-wat**

### CODE REVIEW

**Comment by at-wat** [comment]
@DaikiMaekawa @ykoga-kyutech I'm releasing it to rosdistro. Please take a look.

**Comment by DaikiMaekawa** [comment]
@at-wat Why don't we add LICENSE file?

**Comment by at-wat** [comment]
Ah, let's add LICENSE file.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (DaikiMaekawa) [inline LICENSE]: Fill correct information here.

Comment (at-wat) [reply LICENSE]: This seems instruction to add this to the source code.
google cartographer team is also leaving it as is. https://github.com/googlecartographer/cartographer_ros/blob/master/LICENSE

**Pull request merged**